src/Futhark/CodeGen/ImpGen/GPU.hs

```diff
@@ -225,15 +225,15 @@ withAcc pat inputs lam = do
     locksForInputs atomics ((c, (_, _, op)) : inputs')
       | Just (op_lam, _) <- op,
         AtomicLocking _ <- atomicUpdateLocking atomics op_lam = do
-          let num_locks = 100151
```

Comment:
I think you are messing up this indentation because you are using a slightly too old version of the `ormolu` formatter.